utilities/oadp.py

```
@@ -137,3 +203,136 @@ def create_rhel_vm(
+        wait_complete (bool):
+            Whether to wait for the restore to reach COMPLETED status on context entry.
+        **kwargs (Any):
+            Additional keyword arguments passed to the base ``Restore`` class.
+    """
+
+    def __init__(
+        self,
+        name: str,
+        client: DynamicClient,
+        complete_timeout: int = TIMEOUT_5MIN,
+        wait_complete: bool = True,
```

Ruff (0.14.11)

Boolean-typed positional argument in function definition

(FBT001)